oneflow/core/vm/virtual_machine.cpp

```diff
@@ -411,14 +415,24 @@ Maybe<vm::ThreadCtx*> VirtualMachine::CreateThreadCtx(Symbol<Device> device,
   auto* thread_ctx = *thread_ctx_ptr;
   {
     const auto& WorkerInitializer = [device, stream_type](vm::ThreadCtx* thread_ctx) {
-      int device_type_value = static_cast<int>(device->enum_type());
+      int device_type_value = static_cast<int>(device->enum_type()) * 2
+                              + static_cast<int>(StreamTypeIsTmp::Visit(stream_type));
       CHECK_GT(device_type_value, 0);
-      std::string device_tag = *CHECK_JUST(DeviceTag4DeviceType(device->enum_type()));
+      const std::string thread_tag = [&]() {
+        std::string device_tag = *CHECK_JUST(DeviceTag4DeviceType(device->enum_type()));
+        if (StreamOnIndependentThread::Visit(stream_type)) {
+          return device_tag + GetStreamTypeName::Visit(stream_type);
```

Comment:
```suggestion
          return device_tag + "_" + GetStreamTypeName::Visit(stream_type);
```
这样是不是和下面的 `device_tag + "_tmp"` 更统一

Reply:
已完全重构，没有tmp了。直接是worker线程编号。